store/lib/store/products/product.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Store.Products.Product do
+  use Store.Schema
+
+  import Ecto.Changeset
+
+  schema "products" do
+    field :title, :string
+
+    has_many :taggings, Store.Taggable.Tagging
+    has_many :tags, through: [:taggings, :tag]
+
+    timestamps(type: :utc_datetime)
+  end
+
+  @doc false
+  def changeset(product, attrs) do
+    product
+    |> cast(attrs, [:title])
```

Comment:
It would be great to allow cascading inserts with [`cast_assoc/3`](https://hexdocs.pm/ecto/Ecto.Changeset.html#cast_assoc/3) here.